### PR TITLE
Grab Bag of Mod Support

### PR DESCRIPTION
**Pull request by Sunconure11**
This adds support for Combustive Fishing, NetherEx, Betweenlands, Aether Legacy, and Ice and Fire.

### CODE REVIEW

File: src/main/resources/assets/nutrition/configs/nutrients/protein.json [hunk 1]
```diff
@@ -424,6 +424,8 @@
 			"mob_mash:raw_crocoduck",
 			"natura:edibles:0",
 			"natura:edibles:1",
+			"nex:food_meat_ghast_cooked".
```
Comment (KnightMiner) [inline src/main/resources/assets/nutrition/configs/nutrients/protein.json]: That's a period at the end of the line, should be a comma

Comment (KnightMiner) [reply src/main/resources/assets/nutrition/configs/nutrients/protein.json]: Nevermind, Github was just showing me outdated commits

**Comment by WesCook** [comment]
I know I've been sitting on this one for a while, but I haven't gotten back into the swing of things with mod development just yet.  I'll look into merging it at that point.

In the mean time, thanks for all your contributions.  You've added a lot of mods to the supported list.

**Comment by WesCook** [comment]
It's been a while since I've gone through all the supported mods and made sure they're up-to-date.  I'll do so with this update.  I'm pretty sure Harvestcraft at least has added some new foods since the last release.

**Comment by Sunconure11** [comment]
Salty Mod and Vanilla Food Pantry added some new things too IIRC

**Comment by WesCook** [comment]
These changes have been included in [v2.7.0](https://github.com/WesCook/Nutrition/releases/tag/v2.7.0).

That was a biggie.

**Pull request merged**